php/class-wp-cli.php

```diff
@@ -592,7 +592,8 @@ public static function add_command( $name, $callable, $args = array() ) {
 
 		if ( ! empty( $parent ) ) {
 			$sub_command = trim( str_replace( $parent, '', $name ) );
-			self::debug( "Adding command: {$sub_command} in {$parent} Namespace", 'commands' );
+			self::debug( "Adding namespace: {$parent}", 'commands' );
```

Comment:
This message needs to be displayed when the actual namespace object is being added, otherwise it serves no purpose.

To check whether a command to be added is a namespace, you can use:
```php
if ( $leaf_command instanceof Dispatcher\CommandNamespace ) { }
```

Reply:
Alright, I thought this was week like make none sense what I did, but perhaps with the condition of instanceof perhaps makes more sense. I'll correct it for today.